models/measurements.py

```diff
@@ -40,9 +47,257 @@ class Measurements(Base, AutoIDMixin, SpatiallyIndexed):
         doc="The provenance of this measurement. "
     )
 
-    # TODO: we need to decide what columns are actually saved.
-    #  E.g., should we save a single flux or an array/JSONB of fluxes?
-    #  Same thing for scores (e.g., R/B).
-    #  Are analytical cuts saved with the "scores"?
-    #  What about things like centroid positions / PSF widths?
+    flux_psf = sa.Column(
+        sa.Float,
+        nullable=False,
+        doc="PSF flux of the measurement. "
+    )
+
+    flux_psf_err = sa.Column(
+        sa.Float,
+        nullable=False,
+        doc="PSF flux error of the measurement. "
+    )
+
+    flux_apertures = sa.Column(
+        sa.ARRAY(sa.Float),
+        nullable=False,
+        doc="Aperture fluxes of the measurement. "
+    )
+
+    flux_apertures_err = sa.Column(
+        sa.ARRAY(sa.Float),
+        nullable=False,
+        doc="Aperture flux errors of the measurement. "
+    )
+
+    aper_radii = sa.Column(
+        sa.ARRAY(sa.Float),
+        nullable=False,
+        doc="Radii of the apertures used for calculating flux, in pixels. "
+    )
+
+    best_aperture = sa.Column(
+        sa.SMALLINT,
+        nullable=False,
+        default=-1,
+        doc="The index of the aperture that was chosen as the best aperture for this measurement. "
+            "Set to -1 to select the PSF flux instead of one of the apertures. "
+    )
+
+    mjd = association_proxy('cutouts', 'sources.image.mjd')
+
+    exp_time = association_proxy('cutouts', 'sources.image.exp_time')
+
+    filter = association_proxy('cutouts', 'sources.image.filter')
+
+    @property
+    def mag_psf(self):
+        return -2.5 * np.log10(self.flux_psf) + self.zp.zp  # what about aperture correction?
```

Comment:
psf photometry shouldn't need aperture correction.  (Assuming the zp is a full-psf zp, not an aperture zp... I'm pretty sure we did it that way.  It's the most sane thing to do, but is *not* what the lensgridner pipeline does.)

Reply:
I think our ZP uses a "big" aperture and then adds corrections for the smaller apertures. But will need to check.